lib/ansible/modules/network/avi/avi_controllerproperties.py

```diff
@@ -112,27 +121,42 @@
         description:
             - Period for cluster ip gratuitous arp job.
             - Default value when not specified in API or module is interpreted by Avi Controller as 60.
+        type: int
     consistency_check_timeout_period:
         description:
             - Period for consistency check job.
             - Field introduced in 18.1.1.
             - Default value when not specified in API or module is interpreted by Avi Controller as 60.
         version_added: "2.9"
+        type: int
     crashed_se_reboot:
         description:
             - Number of crashed_se_reboot.
             - Default value when not specified in API or module is interpreted by Avi Controller as 900.
+        type: int
     dead_se_detection_timer:
         description:
             - Number of dead_se_detection_timer.
             - Default value when not specified in API or module is interpreted by Avi Controller as 360.
+        type: int
+    default_minimum_api_timeout:
+        description:
+            - Minimum api timeout value.if this value is not 60, it will be the default timeout for all apis that do not have a specific timeout. if an api has
```

Comment:
```suggestion
            - Minimum API timeout value. If this value is not 60, it will be the default timeout for all APIs that do not have a specific timeout.
```